### DISCUSSION TITLE
anybody makes MyShell works?cannot access result and no error occurs

### DISCUSSION THREAD

**Discussion by yiGmMk** [xtekky/gpt4free#2019] [General]
![image](https://github.com/xtekky/gpt4free/assets/39377339/de20b168-0aa1-4b09-bf44-a3f3404344e4)
when debug with code,I always got Chunk as None here

**Comment by Abdomi12** [comment]
Having trouble with MyShell and unable to access results without any error message? It might be helpful to check your settings or seek technical support. For managing your finances, you can easily access your [FAB bank statement online](https://fabsalarycheck.com/how-to-get-fab-bank-statement-online-guide/) for a clear view of your transactions and account status.
